pkg/manager/member/utils.go

```diff
@@ -176,6 +176,19 @@ func serviceEqual(new, old *corev1.Service) (bool, error) {
 	return false, nil
 }
 
+// isSubMap returns whether the first map is a sub map of the second map
+func isSubMapOf(first map[string]string, second map[string]string) bool {
```

Comment:
What if `first=nil & second !=nil`?
Suggest handling the nil cases before `for`.

Reply:
`for range` a nil map is safe, I add a test case for it:
```
	g.Expect(isSubMapOf(
		nil,
		map[string]string{
			"k1": "v1",
		})).To(BeTrue())
```

Reply:
Yes, sorry, I misunderstood the target of this function.